content/blog/2021-08-24-pre-defined-models-exp.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Using Experiments to Improve Pre-trained Models
+date: 2021-08-24
+description: |
+There are times it will be easier to take a pretrained model and fine-tune it to work with your data. You can do that with DVC experiments.
+descriptionLong: |
+When you're working with a pretrained model and you want to test different fine-tuning values, using DVC experiments can help you do that faster.
+picture: 2021-08-17/pretrained-models.png
+pictureComment: Using Experiments to Improve Pre-trained Models
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/pretrained-model-experiments/727
+tags:
+  - MLOps
+  - Experiments
+  - Reproducibility
+  - DVC
+---
+
+## Intro
+
+There are plenty of machine learning models available that have been trained to
+solve one problem and the knowledge gained from that can be applied to a new,
+yet related problem. For example, a model like AlexNet has been trained on
+millions of images so you could potentially use this to classify cars, animals,
+or even people. This is called transfer learning and it can save a lot of time
+on developing a model from scratch.
+
+For us to take advantage of transfer learning, we can use fine-tuning to adopt
```

Comment:
Should we use this in the title/descriptions? Something "Faster Transfer Learning with Experiments"? (can capture more attention?)

or `Using Experiments for Transfer Learning`

Reply:
Smooth Transfer Learning

or 

Pain-free Transfer Learning :)

Reply:
I like `Using Experiments for Transfer Learning`!